src/main/java/duke/task/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke.task;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+/**
+ * This class implements a storage object that deals with loading tasks from the file and saving tasks in the file.
+ *
+ * CS2103T ip
+ * AY21/22 Semester 1
+ *
+ * @author Kishendran Vendar Kon (Group G05)
+ */
+public class Storage {
+    private File f;
+    private String txtPath;
+
+    /**
+     * Default constructor.
+     *
+     * @param txtFile String representing TextFile used for storage.
+     * @throws IOException if file or directory cannot be created.
+     */
+    public Storage(String txtFile) throws IOException {
+        String home = System.getProperty("user.home");
+        Path path = Paths.get(home, "duke", "dir");
+        txtPath = new StringBuilder().append(home).append("\\duke")
```

Comment:
Doesn't `\\` only apply to Windows (correct me if I'm wrong)? I find that `File.separator` may be useful in this case.